Screenbox/Strings/en-US/Resources.resw

```diff
@@ -803,20 +824,26 @@
   <data name="GetLivelyVisuals" xml:space="preserve">
     <value>Download visuals on GitHub</value>
   </data>
+  <data name="VisualizerOptionsToolTip" xml:space="preserve">
+    <value>Audio visual options</value>
+  </data>
+  <data name="Visualizers" xml:space="preserve">
+    <value>Visuals</value>
+  </data>
   <data name="BrowseFiles" xml:space="preserve">
-    <value>Browse file(s)</value>
+    <value>Browse files</value>
   </data>
   <data name="Default" xml:space="preserve">
     <value>Default</value>
   </data>
   <data name="SettingsAudioVisualDescription" xml:space="preserve">
-    <value>Choose an imagery style to be displayed on audio playback</value>
+    <value>Choose a backdrop to give your audio playback a visual flair in the immersive view</value>
   </data>
   <data name="PoweredByLivelyWallpaper" xml:space="preserve">
     <value>Powered by Lively Wallpaper</value>
   </data>
   <data name="SettingsImportVisualsHeader" xml:space="preserve">
-    <value>Import visuals</value>
+    <value>Import a visual</value>
```

Comment:
The button allows you to select multiple files

Reply:
![Windows 11 background settings snippet](https://github.com/user-attachments/assets/7fc3b895-ea5f-4810-b2eb-4da800ca9bd8)

I took inspiration from the System Settings

Reply:
I see. I have no problem with this then.

Reply:
Already reverted, its fine either way. If you want I can re-revert it?